docs/end-to-end-demo.md

```diff
@@ -58,11 +58,6 @@ carry out one or the other, then continue on with the rest of the steps.
     ```sh
     alias kubectl='microk8s kubectl'
     ```
-1. Install Helm.
-    ```sh
-    sudo apt install -y curl
-    curl -L https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
-    ```
 1. Enable Helm for MicroK8s.
     ```sh
     kubectl config view --raw >~/.kube/config
```

Comment:
I walked through these steps on a fresh Ubuntu 20.04 VM, and it looks like this is not necessary when enabling Helm through MicroK8s, so I think you can remove it. @bfjelds, any reason we should keep this (`kubectl config view --raw >~/.kube/config`)?

Reply:
i'm say remove it if it isn't needed!

Reply:
should we adjust our e2e tests with this as well?

Reply:
(maybe as a separate PR for the test changes)

Reply:
Totally agree. lets change the tests (in a separate PR).

Reply:
Fixed: https://github.com/deislabs/akri/pull/55/commits/680f9fdc9d3bd322d859d2c6c5cf3aa2f594b14a